css-three-column-layout/styles.css

```diff
@@ -0,0 +1,69 @@
+body {
+  background-color: #0090e0;
+  font-family: Lato, Arial, Helvetica, sans-serif;
+}
+
+.container {
+  max-width: 850px;
+  margin: auto;
+  background-color: #0090e0;
+  color: white;
+}
+
+.header {
+  margin-top: 70px;
+  display: flex;
+  flex-basis: 100%;
+}
+
+.white {
+  height: 100px;
+  width: 100px;
+  margin: 44px 33px 33px 96px;
+  border-radius: 50%;
+  background-color: rgb(255 255 255);
+  position: absolute;
+}
+
+.column-one {
```

Comment:
column classes should have widths explicitly set. your current setup is taking advantage of the fact that display flex on the row class automatically creates even sized child elements, but that should be avoided. Always make sure to set an actual width on a column class.